mirroroperator/registrymirror.py

```diff
@@ -20,18 +27,23 @@ def __init__(self, event_type, namespace, hostess_docker_registry,
         self.hostess_docker_image = hostess_docker_image
         self.hostess_docker_tag = hostess_docker_tag
         self.docker_certificate_secret = docker_certificate_secret
-        self.kind = kwargs.get("kind")
```

Comment:
Can we split out these refactoring changes separate to the feature? (As per our standard on commits)

Reply:
(Same goes for the other lines that are just refactoring)

Reply:
Also the same for the formatting changes

Reply:
ok, I think I have done this everywhere